docs/build/guides/account-linking/child-accounts.md

```diff
@@ -151,87 +155,91 @@ transaction(identifiers: [String]) {
+            acct.capabilities.storage.issue<&CapabilityFactory.Manager>(CapabilityFactory.StoragePath),
+            at: CapabilityFactory.PublicPath
+        )
 
         assert(
-            acct.getCapability<&CapabilityFactory.Manager{CapabilityFactory.Getter}>(CapabilityFactory.PublicPath).check(),
+            acct.capabilities.get<&CapabilityFactory.Manager>(CapabilityFactory.PublicPath).check(),
             message: "CapabilityFactory is not setup properly"
         )
 
-        let manager = acct.borrow<&CapabilityFactory.Manager>(from: CapabilityFactory.StoragePath)
+        let manager = acct.storage.borrow<auth(CapabilityFactory.Add) &CapabilityFactory.Manager>(from: CapabilityFactory.StoragePath)
             ?? panic("manager not found")
 
         /// Add generic NFT-related Factory implementations to enable castable Capabilities from this Manager
-        manager.addFactory(Type<&{NonFungibleToken.CollectionPublic}>(), NFTCollectionPublicFactory.Factory())
-        manager.addFactory(Type<&{NonFungibleToken.Provider, NonFungibleToken.CollectionPublic}>(), NFTProviderAndCollectionFactory.Factory())
-        manager.addFactory(Type<&{NonFungibleToken.Provider}>(), NFTProviderFactory.Factory())
+        manager.updateFactory(Type<&{NonFungibleToken.CollectionPublic}>(), NFTCollectionPublicFactory.Factory())
+        manager.updateFactory(Type<auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Provider, NonFungibleToken.CollectionPublic}>(), NFTProviderAndCollectionFactory.Factory())
+        manager.updateFactory(Type<auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Provider}>(), NFTProviderFactory.Factory())
+        manager.updateFactory(Type<auth(NonFungibleToken.Withdraw) &{NonFungibleToken.Collection}>(), NFTCollectionFactory.WithdrawFactory())
+        manager.updateFactory(Type<&{NonFungibleToken.Collection}>(), NFTCollectionFactory.Factory())
     }
 }
 ```
 
 ![resources/hybrid_custody_high_level](./resources/hybrid_custody_high_level.png)
 
-*In this scenario, a user custodies a key for their main account which maintains access to a wrapped AuthAccount
+_In this scenario, a user custodies a key for their main account which maintains access to a wrapped `Account`
 Capability, providing the user restricted access on the app account. The app maintains custodial access to the account
-and regulates the access restrictions to delegatee "parent" accounts.*
+and regulates the access restrictions to delegatee "parent" accounts._
 
-Linking accounts can be done in one of two ways. Put simply, the child account needs to get the parent account an
-`AuthAccount` Capability, and the parent needs to save that Capability so they can retain access in a manner that also
-represents each side of the link and safeguards the integrity of any access restrictions an application puts in place on
-delegated access.
+Linking accounts can be done in one of two ways. Put simply, the child account needs to get the parent an `Account`
```

Comment:
I'm not sure either since you never really deal with an `Account`, it's always a reference. It's probably more consistent to update these instances to `&Account`

Reply:
yeah, I think I agree